index.html

```diff
@@ -188,20 +188,22 @@ <h2 id="basic-usage-cdn">Usage with CDNs</h2>
 	<p>In combination with CDNs, we recommend using the <a href="plugins/autoloader">Autoloader plugin</a> which automatically loads languages when necessary.</p>
 
 	<p>The setup of the Autoloader, will look like the following. You can also add your own themes of course.</p>
-
+		
 	<pre><code>&lt;!DOCTYPE html>
 &lt;html>
 &lt;head>
 	...</code>
-	<code class="highlight">&lt;link href="https://my.cdn/prism@v1.x/themes/prism.css" rel="stylesheet" /></code>
+	<code class="highlight">&lt;link href="https://example.com/prism@v1.x/themes/prism.css" rel="stylesheet" /></code>
```

Comment:
Could we use `cdn.example.com` instead? IMO, it's not clear that `example.com` is supposed to be a CDN.